lib/tiled/map.rb

```diff
@@ -89,6 +89,28 @@ def pixelheight
       return height * tileheight
     end
 
+    #gets the first object layer with the passed name of `layer_name`
```

Comment:
I would prefer to format comments in yard syntax, you can check the example in the `lib/tiled/layers.rb` file.